tests/test_modeling_transfo_xl.py

```diff
@@ -304,6 +304,14 @@ def test_resize_embeddings_untied(self):
         # transfo-xl requires special resize for lm-head
         return
 
+    def _check_attentions_for_generate(self, *args, **kwargs):
+        # TODO(PVP)
+        pass
+
+    def _check_hidden_states_for_generate(self, *args, **kwargs):
```

Comment:
xlnet has dual-stream architecture which leads to special attentions and hidden_states